.github/actions/assemble-devnet/action.yml

```diff
@@ -66,9 +66,9 @@ runs:
       run: |
         cd /tmp/
         echo "downloading linea-besu: ${{ steps.dotenv.outputs.LINEA_BESU_TAR_GZ }}"
-        wget -nv "https://artifacts.consensys.net/public/linea-besu/raw/names/linea-besu.tar.gz/versions/${{ steps.dotenv.outputs.LINEA_BESU_TAR_GZ }}/linea-besu-${{ steps.dotenv.outputs.LINEA_BESU_TAR_GZ }}.tar.gz"
-        tar -xvf linea-besu-${{ steps.dotenv.outputs.LINEA_BESU_TAR_GZ }}.tar.gz
-        mv /tmp/linea-besu-${{ steps.dotenv.outputs.LINEA_BESU_TAR_GZ }} /tmp/besu
+        wget -nv "https://github.com/Consensys/linea-besu-upstream/releases/download/${{ steps.dotenv.outputs.LINEA_BESU_TAR_GZ }}/besu-${{ steps.dotenv.outputs.LINEA_BESU_TAR_GZ }}.tar.gz"
```

Comment:
Since we are not going to update all the envs at the same time, it is better to put this and other changing strings in env variables file, so we can deploy on env at time

Reply:
done.